docs/kubernetes/kind.mdx

```diff
@@ -0,0 +1,185 @@
+
+Before installing the operator, it is necessary to install two prerequisites:
+
+- A **[NATS](/docs/ecosystem/nats/) cluster** (with Jetstream enabled)
+- **[wasmCloud Application Deployment Manager (wadm)](/docs/ecosystem/wadm/)** 
+
+The following steps will help you deploy the prerequisites to your Kubernetes cluster, then deploy the operator.
+
+### Deploy NATS
+
+NATS is the open source connective technology that wasmCloud uses to create a unified topology across any number of environments. Add the NATS Helm repository:
+
+```shell
+helm repo add nats https://nats-io.github.io/k8s/helm/charts/
+```
+
+Install the upstream NATS Helm chart to start a cluster with the `values.yaml` file from the wasmCloud operator repository:
+
+```shell
+helm upgrade --install -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/nats-values.yaml nats nats/nats
+```
+
+Validate the installation with:
+
+```shell
+kubectl rollout status deploy,sts -l app.kubernetes.io/instance=nats
+```
+
+#### Deploy wadm
+
+wasmCloud Application Deployment Manager (wadm) enables declarative application deployment and management for wasmCloud. You can deploy wadm to your Kubernetes cluster with a Helm chart:
+
+```shell
+helm install wadm -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/wadm-values.yaml oci://ghcr.io/wasmcloud/charts/wadm
+```
+
+Validate the installation with:
+
+```shell
+kubectl rollout status deploy -l app.kubernetes.io/instance=wadm
+```
+
+#### Deploy the wasmCloud operator
+
+To deploy the operator:
+
+```shell
+kubectl apply -k https://github.com/wasmCloud/wasmcloud-operator/deploy/base
+```
+
+Validate the installation with:
+
+```shell
+# make sure pods are ready
+kubectl rollout status deploy -l app=wasmcloud-operator -n wasmcloud-operator
+# apiservice should be available
+kubectl get apiservices.apiregistration.k8s.io v1beta1.core.oam.dev
+```
+
+### Create wasmCloud host
+
+Apply the `wasmcloud-host` manifest:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/wasmcloud-host.yaml
+```
+
+Check wasmCloud host status:
+
+```shell
+kubectl describe wasmcloudhostconfig wasmcloud-host
+```
+
+### Run a WebAssembly component on Kubernetes
+
+When you `kubectl apply` a [wasmCloud application manifest](../../docs/ecosystem/wadm/model), the cluster automatically provisions the component workload with wasmCloud. 
+
+This example uses the [`hello-world-application.yaml` manifest](https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/hello-world-application.yaml) included in the operator's quickstart. (The [source code for the application](https://github.com/wasmCloud/wasmCloud/tree/main/examples/rust/components/http-hello-world) is available in the wasmCloud repository.)
```

Comment:
We should use an example that uses the daemonscaler to workaround this: https://github.com/wasmCloud/wasmCloud-operator?tab=readme-ov-file#automatically-syncing-kubernetes-services